### PR TITLE
Implemented: App Version Info for centralized use (#142)

### LINKED ISSUE #142: Implement Centralized App Version Information
This feature aims to provide users and administrators with a unified and efficient method for accessing and tracking information about the current app version, release notes, and update history. Develop a dedicated Version Information component that serves as a central hub for users and administrators to access details about the current app version. This component should be accessible from a prominent location within the application.

### PR DESCRIPTION
**Pull request by sanskar345**
Closes #142 

### Steps to verify --
- Proper working App Version Info in apps 

Changes made in picking for testing this, please review this also -  https://github.com/hotwax/picking/pull/162

### CODE REVIEW

File: src/components/index.ts [hunk 1]
```diff
@@ -14,5 +14,8 @@ import '@ionic/vue/css/text-transformation.css';
 import '@ionic/vue/css/flex-utils.css';
 import '@ionic/vue/css/display.css';
 
-export { default as ProductIdentifier } from "./ProductIdentifier.vue";
+import AppVersionInfo from './AppVersionInfo.vue';
+import ProductIdentifier from "./ProductIdentifier.vue";
```
Comment (k2maan) [inline src/components/index.ts:18]: Update import export syntax

File: src/components/AppVersionInfo.vue [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<template>
+  <div class="section-header">
+    <h1>
+      {{ translate('App') }}
+      <p class="overline">{{ "Version: " + appVersion }}</p>
+    </h1>
```
Comment (k2maan) [inline src/components/AppVersionInfo.vue:6]: We should not use p tag inside h1.

File: src/components/AppVersionInfo.vue [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<template>
+  <div class="section-header">
+    <h1>
+      {{ translate('App') }}
```
Comment (k2maan) [inline src/components/AppVersionInfo.vue:4]: Replace with $t.

**Review by k2maan — CHANGES_REQUESTED** [review]

**Review by k2maan — APPROVED** [review]

File: src/index.ts [hunk 4]
```diff
@@ -42,11 +42,14 @@ export let dxpComponents = {
     app.use(pinia);
     app.use(i18n);
 
+    app.component('AppVersionInfo', AppVersionInfo)
     app.component('LanguageSwitcher', LanguageSwitcher)
     app.component('Login', Login)
     app.component('ShopifyImg', ShopifyImg)
     app.component('ProductIdentifier', ProductIdentifier)
 
+    translate = options.translate;
```
Comment (ymaheshwari1) [inline src/index.ts:51]: I don't think this is needed.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/components/AppVersionInfo.vue [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    <p class="overline">{{ "Built: " + getDateTime(appInfo.builtTime) }}</p>
+  </div>
+</template>
+  
+<script setup lang="ts">
+import { DateTime } from 'luxon';
+
+declare var process: any;
+
+const appInfo = (process.env.VUE_APP_VERSION_INFO ? JSON.parse(process.env.VUE_APP_VERSION_INFO) : {}) as any;
```
Comment (ymaheshwari1) [inline src/components/AppVersionInfo.vue:16]: We should move the logic to use the `app-version-info` package in dxp itself.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

**Review by ymaheshwari1 — APPROVED** [review]

File: src/components/AppVersionInfo.vue (hunk 5 above)
Comment (sanskar345) [reply src/components/AppVersionInfo.vue:16]: We cannot use the App Version Info package in dxp as per my findings if we use app version info in dxp, we cannot access the env of appInfo created during the app version info Package run in the component. Also, I think we need App Version Info to execute commands during the app build. But as with this the git command will execute during dxp build and the info will be of no use.

**Pull request merged**